src/main/frontend/components/sidebar.cljs

```diff
@@ -142,9 +142,11 @@
        [:ul.favorites.text-sm
         (for [name favorites]
           (when-not (string/blank? name)
-            (when-let [entity (db/entity [:block/name (util/safe-page-name-sanity-lc name)])]
-              (let [icon (get-page-icon entity)]
-                (favorite-item t name icon)))))]))))
+            (let [source-page (db-model/get-alias-source-page (state/get-current-repo) name) ]
+              (let [source-page-name (if (empty? source-page) name (db-model/get-original-name source-page))]
```

Comment:
This `let` expression can be wrapped into the first one.

Reply:
reverted